sdk/search/azure-search-documents/README.md

```diff
@@ -75,7 +75,7 @@ add the direct dependency to your project as follows.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-search-documents</artifactId>
-    <version>11.7.0-beta.3</version>
+    <version>11.7.0-beta.4</version>
```

Comment:
We should not update this version as we are not releasing 11.7.0-beta.4 (this PR is for GA release).

Reply:
Yes, this will get updated before the release to 11.7.0, leaving it as a beta version for now as that release will happen in about two weeks.